src/multivariates.jl

```diff
@@ -121,6 +121,13 @@ Compute the vector of element-wise variances for distribution `d`.
 """
 var(d::MultivariateDistribution)
 
+"""
+    std(d::MultivariateDistribution)
+
+Compute the vector of element-wise standard deviations for distribution `d`.
+"""
+std(d::MultivariateDistribution) = sqrt.(var(d))
```

Comment:
Maybe we could use
```suggestion
function std(d::MultivariateDistribution)
    v = var(d)
    map!(sqrt, v, v)
    return v
end
```
to avoid the broadcast machinery and reduce allocations?